README.md

```diff
@@ -449,13 +460,13 @@ Name | Type |
 
 ### ▪ **validators**: *object*
 
-*Defined in [validators/validators.ts:21](https://github.com/openmrs/openmrs-esm-module-config/blob/9766947/src/validators/validators.ts#L21)*
+*Defined in [validators/validators.ts:21](https://github.com/donaldkibet/openmrs-esm-module-config/blob/0f99e76/src/validators/validators.ts#L21)*
 
 ###  isBoolean
 
 • **isBoolean**: *function*
 
-*Defined in [validators/validators.ts:21](https://github.com/openmrs/openmrs-esm-module-config/blob/9766947/src/validators/validators.ts#L21)*
+*Defined in [validators/validators.ts:21](https://github.com/donaldkibet/openmrs-esm-module-config/blob/0f99e76/src/validators/validators.ts#L21)*
```

Comment:
I guess all of them ran automatically, however, I think we should keep them from the `openmrs` organization (primary) and not from the fork (secondary).

Any other thoughts here? (cc @brandones )

Reply:
yeap it ran automatically fixed it